components/NavBar.js

```diff
@@ -233,25 +232,22 @@ const NavBar = () => {
   };
 
   const handleScroll = () => {
-    let lastScroll = 0
+    var lastScroll = 0;
```

Comment:
Just wanted to double-check if var is really needed here – seems like `lastScroll` is only used within scope. Correct me if I'm wrong though!

Reply:
var is outdated - I'll change it to let